daq/topology.py

```diff
@@ -488,14 +496,23 @@ def _add_dot1x_incoming_rule(self, incoming_acl, secondary_acl):
         for devices in self._set_devices.values():
             for device in devices:
                 if device and device.gateway:
-                    vlan = self._get_port_vlan(device.port.port_no)
+                    vlan = device.vlan if device.vlan else self._get_port_vlan(device.port.port_no)
                     test_ports = device.gateway.get_possible_test_ports()
                     if test_ports:
                         self._add_dot1x_allow_rule(incoming_acl, test_ports, vlan_vid=vlan)
                     device_port = device.port.port_no
                     if device_port:
                         self._add_dot1x_allow_rule(secondary_acl, [device_port], in_vlan=vlan)
 
+    def _add_dot1x_vxlan_acl(self, acls):
+        for devices in self._set_devices.values():
+            for device in devices:
+                if device and device.vlan:
+                    acl_name = 'vxlan_coupler_%s' % device.vlan
```

Comment:
something seems mismatched here since the function name is vxlan_acl but this is adding a coupler-specific acl?

Reply:
Changed method name to reflect coupler_acl instead.